client/app/index.js

```diff
@@ -26,6 +34,8 @@ function loadCurrentModule() {
 }
 
 function loadModules() {
+  loadAceEditorDependencies();
```

Comment:
I think it's better to group them together in `laouy_ace` then, cause `index` is an entry file and we don't `export` method here.